src/sentry/templates/sentry/layout.html

```diff
@@ -15,9 +15,8 @@
   <meta name="robots" content="NONE,NOARCHIVE">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <link href="{% absolute_asset_url "sentry" "images/favicon.ico" %}" rel="shortcut icon" type="image/png"/>
+  <link rel="icon" class="js-site-favicon" type="image/png" href="{% absolute_asset_url "sentry" "images/favicon.png" %}">
 
-  <link rel="icon" href="{% absolute_asset_url "sentry" "images/logos/apple-touch-icon.png" %}">
```

Comment:
This line I think was incorrect and is why sometimes you might get the apple touch icon as your favicon